src/Dockerfile

```diff
@@ -11,7 +11,8 @@ COPY . $WORKDIR
 # installing dependencies to build package
 RUN pip install . -t python
 
-# change 31816
+# change 31337.1
```

Comment:
Any chance of adding to this comment to explain what 'change 31337.1' means?  What does it refer to in the file?  Does it apply to the following line in some way?

Reply:
It's a nonsense change to change the hash of the Dockerfile, which triggers Terraform to replace the resource (otherwise it doesn't). The real fix is to have Terraform look at the hash of more than the Dockerfile, but that requires time. There's a ticket for it: https://gfw.atlassian.net/browse/GTC-1984
Would you mind if I make the suggested change (an explanatory comment) in a follow-up PR as it's so minor and this is already merged to develop?

Reply:
Sure, you can add the comment in some later PR.  Thanks!